### ISSUE TITLE
Make lambdaj available in a maven repository

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dschulten/lambdaj#7]
```
Hi,

Nice project of yours!
It would be nice if lamdaj would be available in a maven repository. You
can create a repository using the googlecode svn server because it is
accessible via HTTP (http://lambdaj.googlecode.com/svn/). You can use the
wagon-svn (https://wagon-svn.dev.java.net/) to deploy the artifacts with maven.
```

Original issue reported on code.google.com by `lehph...@gmail.com` on 23 May 2009 at 2:07

**Comment by GoogleCodeExporter** [comment]
Original comment by `mario.fu...@gmail.com` on 26 Jun 2009 at 7:12
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Hi,

I will suggest you looking at
[http://www.sonatype.com/people/2009/06/publishing-your-artifacts-to-the-central
-maven-repository/
Publishing Your Artifacts to the Central Maven Repository by Brian Fox] to have
public nexus repository synced with maven central.

This project is very usefull for me, a big kudo for you.
```

Original comment by `filosgan...@gmail.com` on 27 Jun 2009 at 1:05

**Comment by GoogleCodeExporter** [comment]
```
Mario, what is the status of this? Releasing lambdaj into a maven repository on 
this
project's svn should be easy. Having it published to maven central is of course
another story.
```

Original comment by `sebastian.jancke@googlemail.com` on 1 Jul 2009 at 7:24

**Comment by GoogleCodeExporter** [comment]
Original comment by `Christia...@gmail.com` on 1 Jul 2009 at 10:58
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
I will try to get a place on the sonatype repository which is synced with the 
maven
central.

I also will update the pom, so that it is releasable (eg. gpg).
```

Original comment by `Christia...@gmail.com` on 1 Jul 2009 at 3:06

**Comment by GoogleCodeExporter** [comment]
```
Any news about this issue?

Thanks
```

Original comment by `federico...@gmail.com` on 14 Jul 2009 at 12:56

**Comment by GoogleCodeExporter** [comment]
```
I did the configurations to deploy to sonatype repo in maven-config-processor
(http://code.google.com/p/maven-config-processor-plugin/), I've found this link
helpful: http://happygiraffe.net/blog/2009/06/24/gnupg-very-basically/
```

Original comment by `lehph...@gmail.com` on 14 Jul 2009 at 1:23

**Comment by GoogleCodeExporter** [comment]
```
As far as I know, the changes have been completed and committed by Christian 
Raschka.
Our own Maven Repository for releases is here:
http://code.google.com/p/lambdaj/source/browse/repo/releases/

For snapshots, we will use:
http://code.google.com/p/lambdaj/source/browse/repo/snapshots/

Currently, publishing to sonatype is in progress. Until then, please use our own
repository as stated above.
```

Original comment by `sebastian.jancke@googlemail.com` on 14 Jul 2009 at 2:39

**Comment by GoogleCodeExporter** [comment]
```
Paths are wrong. Correct ones are:
http://lambdaj.googlecode.com/svn/repo/releases/

and
http://lambdaj.googlecode.com/svn/repo/snapshots/
```

Original comment by `sebastian.jancke@googlemail.com` on 16 Jul 2009 at 8:00

**Comment by GoogleCodeExporter** [comment]
```
Thanks!
```

Original comment by `lehph...@gmail.com` on 16 Jul 2009 at 1:44

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Could you please add the code and javadoc in the repo. This is very useful when 
it
comes to debugging (even if not in your library).
This is quite easy to do, just adding two build plugins in your pom.xml should 
be enough:

      <plugin>
        <groupId>org.apache.maven.plugins</groupId>
        <artifactId>maven-source-plugin</artifactId>
        <executions>
          <execution>
            <id>attach-sources</id>
            <goals>
              <goal>jar</goal>
            </goals>
          </execution>
        </executions>
      </plugin>
      <plugin>
        <groupId>org.apache.maven.plugins</groupId>
        <artifactId>maven-javadoc-plugin</artifactId>
        <executions>
          <execution>
            <id>attach-javadocs</id>
            <goals>
              <goal>jar</goal>
            </goals>
          </execution>
        </executions>
      </plugin>

Thanks

```

Original comment by `jose.chi...@gmail.com` on 1 Sep 2009 at 11:05

**Comment by GoogleCodeExporter** [comment]
```
I agree, this should be added. Also assembly:jar-with-dependencies should be 
added to
the repo (as it is published by google downloads already!).
```

Original comment by `sebastian.jancke@googlemail.com` on 3 Sep 2009 at 4:43

**Comment by GoogleCodeExporter** [comment]
```
Will be available from lambdaj 2.1
```

Original comment by `mario.fu...@gmail.com` on 10 Oct 2009 at 4:17
- Changed state: **Done**
